project/frontend/src/components/ResultsMap.js

```diff
@@ -8,6 +8,46 @@ function MapContainer(props) {
 
   const MarkerIcon = () => <img src={Lunch} alt='Lunch icon' />;
 
+  const InfoWindow = (props) => {
+    // const { place } = {
+    //   name: 'yah',
+    //   rating: 2.5,
+    //   opening_hours: { open_now: true },
+    //   types: ['Italian'],
+    // };
+    const infoWindowStyle = {
+      position: 'relative',
+      bottom: 150,
+      left: '-45px',
+      width: 220,
+      backgroundColor: 'white',
+      boxShadow: '0 2px 7px 1px rgba(0, 0, 0, 0.3)',
+      padding: 10,
+      fontSize: 14,
+      zIndex: 100,
+    };
+
+    return (
+      <div style={infoWindowStyle}>
+        <div style={{ fontSize: 16 }}>{'Applebees'}</div>
+        <div style={{ fontSize: 14 }}>
+          <span style={{ color: 'grey' }}>{2.5} </span>
+          <span style={{ color: 'orange' }}>
+            {String.fromCharCode(9733).repeat(Math.floor(2.5))}
```

Comment:
Just curious what this and line 39 are doing (something with the stars?)

Reply:
if you look on the card the String.fromCharCode(9733) produces the star so we want the floored rating to have orange stars and the rest of them to be gray. I'm about to push code with var names so certain parts of the card become more clear!